ds_autotuning_prototype/trial_classes/cifar10_moe_fake_data/ds_config.json

```diff
@@ -0,0 +1,43 @@
+{
+  "train_batch_size": 16,
+  "steps_per_print": 2000,
+  "optimizer": {
+    "type": "Adam",
+    "params": {
+      "lr": 0.001,
+      "betas": [0.8, 0.999],
+      "eps": 1e-8,
+      "weight_decay": 3e-7
+    }
+  },
+  "scheduler": {
+    "type": "WarmupLR",
+    "params": {
+      "warmup_min_lr": 0,
+      "warmup_max_lr": 0.001,
+      "warmup_num_steps": 1000
+    }
+  },
+  "gradient_clipping": 1.0,
+  "prescale_gradients": false,
+  "fp16": {
+    "enabled": true,
+    "fp16_master_weights_and_grads": false,
+    "loss_scale": 0,
+    "loss_scale_window": 500,
+    "hysteresis": 2,
+    "min_loss_scale": 1,
+    "initial_scale_power": 15
+  },
+  "wall_clock_breakdown": false,
+  "zero_optimization": {
```

Comment:
Question: In this case, if we use the autotuning, is this optimization dictionary ignored?

Reply:
Depends, but potentially, yes.

With this PR you can just search over a subspace of the available `zero_optimization` keys. So, like you could only search over `reduce_bucket_size` and `dsat` will overwrite that field, but all of the other values in the present config would remain as they are here.